addons/l10n_it_edi/models/account_invoice.py

```diff
@@ -134,10 +134,6 @@ def _l10n_it_edi_prepare_fatturapa_tax_details(self, tax_details, reverse_charge
                     tax_dict['rounding'] = base_amount - (tax_amount * 100 / tax_rate)
                     tax_dict['base_amount'] = base_amount - tax_dict['rounding']
 
-            if not reverse_charge_refund:
-                tax_dict['base_amount'] = abs(tax_dict['base_amount'])
-                tax_dict['tax_amount'] = abs(tax_dict['tax_amount'])
-
```

Comment:
Are you sure? As far as I can see, the parameter reverse_charge_refund was used in the case of bill refund which you don't seem to test anymore.

```python
215         # Represent if the document is a reverse charge refund in a single variable
216         reverse_charge = document_type in ['TD17', 'TD18', 'TD19']
217         is_downpayment = document_type in ['TD02']
218         reverse_charge_refund = self.move_type == 'in_refund' and reverse_charge
```

Also, please note that during the forward-porting to master, you may want to completely remove the reverse_charge_refund kwarg, since it's now deadcode

Reply:
Actually, we are still testing it, just in a different test file :smile: 

https://github.com/odoo/odoo/blob/b72dc83b6050c6b662d6d85ebb94cdb9709aa433/addons/l10n_it_edi/tests/test_edi_reverse_charge.py#L255-L289

This PR had indeed broken that test, and it wasn't being run on runbot because the l10n tests aren't run on community PRs. I'll fix it.